variables.md

```diff
@@ -195,7 +195,8 @@ QAM_MINIMAL | string | "full" or "small" | Full is adding patterns x11, gnome-ba
 RAIDLEVEL | integer | | Define raid level to be configured. Possible values: 0,1,5,6,10.
 REBOOT_TIMEOUT | integer | 0 | Set and handle reboot timeout available in YaST installer. 0 disables the timeout and needs explicit reboot confirmation.
 REGISTRY | string | docker.io | Registry to pull third-party container images from
-CONTAINER_IMAGE_VERSIONS | string | | List of comma-separated versions from `get_suse_container_urls()` to test in `tests/containers/*_image.pm`
+CONTAINER_IMAGE_VERSIONS | string | | List of comma-separated versions from `get_suse_container_urls()`
+CONTAINER_IMAGES_TO_TEST | string | | The URL of a specific container image to test.
```

Comment:
My initial idea of this variable was for single image instead of array, as you describe it here, so it would be `CONTAINER_IMAGE_TO_TEST`.
But in the code `get_urls_from_var()` you are giving the possibility to provide an array, which is not a bad idea either. If we go the "array" way, we should also mention it here, e.g. `List of comma-separated URLs of container images to be tested`

Reply:
I decided to go this way because of if `tests/containers/image.pm` we have:
```perl
for my $iname (@{$images_to_test}) {
```
So even if I keep `CONTAINER_IMAGE_TO_TEST` I'd need to translate it to array.

Reply:
Fixed.